packages/react/src/components/table/table.tsx

```diff
@@ -72,6 +73,34 @@ function getHeading<T extends object>(column: HeaderGroup<T>, stickyHeader: bool
     );
 }
 
+const StyledFooter = styled.td<{ sticky: boolean }>`
+    ${({ sticky }) => sticky && css`
+        background-color: ${({ theme }) => theme.greys.white};
+        position: sticky;
+    `}
+`;
+
+function getFooter<T extends object>(column: HeaderGroup<T>, stickyFooter: boolean): ReactElement | null {
+    if (column.Footer === '') {
+        // If the Footer content is empty, skip rendering the td
+        return null;
+    }
+    const footerProps = column.colSpan
```

Comment:
Not ideal here because we're getting a `HeaderGroup` from `react-table` but we're using it as a our own `CustomColumn`. It works at runtime but it's not type-safe.

(I know that's what we did for the header, I still don't like it too much).

Reply:
Updated the code. gonna wait for your feedback as our convo.
Thanks.

Reply:
fixed based on @meriouma feedback.